personhood/api.go

```diff
@@ -20,12 +23,48 @@ func NewClient() *Client {
 	return &Client{Client: onet.NewClient(cothority.Suite, ServiceName)}
 }
 
-// LinkPoP sends a party description to the message server for further
-// reference in messages.
-func (c *Client) LinkPoP(si *network.ServerIdentity, p Party) error {
-	err := c.SendProtobuf(si, &LinkPoP{p}, nil)
-	if err != nil {
-		return err
+// TestData stores a byzcoin-ID and a spawner-IID in the system for the other clients to pick up. This is a
+// test-endpoint, not used in the final personhood.online app.
+func (c *Client) TestData(r onet.Roster, bcID skipchain.SkipBlockID, spawnIID byzcoin.InstanceID) (errs []error) {
```

Comment:
I use it in the personhood.online app to do UI testing. When you start the app in testing-mode, you can create your own byzcoin blockchain, including an admin account, and do some mining of coins. This makes it much easier to quickly launch three emulators, create a chain, and onboard the other two emulators.

If you have a better idea how this could be done, I'm all ears.